db/migrate/20220513101329_create_airlines.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAirlines < ActiveRecord::Migration[7.0]
+  def change
+    create_table :airlines do |t|
+      t.string :name
+      t.string :image_url
+      t.string :slug
```

Comment:
Change all the `string` to `text` because it may exceed 255 characters which is the limit of string.

Note: `string` == `varchar` in rdbms terms